relay-server/src/actors/project_buffer.rs

```diff
@@ -243,15 +243,19 @@ impl BufferService {
                 .shared_cache(true);
 
             let db = SqlitePoolOptions::new()
-                .max_connections(config.cache_persistent_buffer_max_connections())
-                .min_connections(config.cache_persistent_buffer_min_connections())
+                .max_connections(config.spool_envelopes_max_connections())
+                .min_connections(config.spool_envelopes_min_connections())
                 .connect_with(options)
                 .await?;
 
             let spool_config = BufferSpoolConfig {
                 db,
-                max_disk_size: config.cache_persistent_buffer_max_disk_size(),
-                max_memory_size: config.cache_persistent_buffer_max_memory_size(),
+                max_disk_size: config.spool_envelopes_max_disk_size(),
+                // It is a rough extimation for how many envelopes we can fit in the
+                // configured memory limit, taking that 1 enveloper is 1 MB.
+                //
+                // TODO: Can we calculate the size of the envelope?
+                max_envelopes_count: config.spool_envelopes_max_memory_size() / (1024 * 1024),
```

Comment:
Something like this?

```suggestion
                max_envelopes_count: config.spool_envelopes_max_memory_size() / config.max_envelope_size(),
```

Reply:
done in [15f57c7](https://github.com/getsentry/relay/pull/2012/commits/15f57c7a6b6d26639f69f7c74c864e2ff441c495)